modin/backends/pandas/parsers.py

```diff
@@ -11,30 +11,87 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
+"""This module houses parser classes (classes that are used for data parsing on the workers)
+and util functions for handling parsing results.
+
+"""
+
 from collections import OrderedDict
 from io import BytesIO
 import numpy as np
 import pandas
 from pandas.core.dtypes.cast import find_common_type
 from pandas.core.dtypes.concat import union_categoricals
 from pandas.io.common import infer_compression
+from pandas.util._decorators import Appender
 import warnings
 
 from modin.engines.base.io import FileDispatcher
 from modin.data_management.utils import split_result_of_axis_func_pandas
 from modin.error_message import ErrorMessage
 
 
+_doc_pandas_parser_class = """"Class for handling {data_type} on the workers
+using Pandas backend, inherits common functions from `PandasParser` class.
+
+"""
+
+_doc_parse_parameters_common = """fname: str or path object
+    Name of the file or path to read.
+"""
+
+_doc_parse_parameters_csv_glob = """chunks: list
+    List, where each element of the list is a list of tuples. The inner lists
+    of tuples contains the data file name of the chunk, chunk start offset, and chunk end offsets for its corresponding file.
+"""
+
+_doc_parse_parameters_hdf = """fname: str, path object, pandas.HDFStore or file-like object
+    Name of the file, path pandas.HDFStore or file-like object to read.
+"""
+
+_doc_parse_parameters_feather = """fname: str, path object or file-like object
+    Name of the file, path or file-like object to read.
+"""
+
+_doc_parse_parameters_sql = """sql: string or SQLAlchemy Selectable (select or text object)
```

Comment:
I wonder if we should just inline these parameter descriptions - each is used once (at least that's what I saw)

Reply:
Fixed.